### PR TITLE
Add a migration for jpeg to piggyback on it

### PR DESCRIPTION
**Pull request by hmaarrfk**
xref: https://github.com/conda-forge/conda-forge.github.io/issues/673
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/migrations/jpeg9e.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+migrator_ts: 1676157958
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  # limit the number of prs to do the initial test
+  pr_limit: 1
```
Comment (isuruf) [inline recipe/migrations/jpeg9e.yaml:7]: Please add a custom PR title here.

Comment (hmaarrfk) [reply recipe/migrations/jpeg9e.yaml:8]: Via the commit message? or is there an other key.

File: recipe/migrations/jpeg9e.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  migration_number: 1
+  bump_number: 1
+  commit_message: "Rebuild for jpegturbo migration"
+  # limit the number of prs to do the initial test
+  pr_limit: 1
+
+# This migration is matched with a minimigrator that will
+# replace jpeg with libjpegturbo
+jpeg:
+  - '9e'
```
Comment (isuruf) [inline recipe/migrations/jpeg9e.yaml:13]: Are you sure `9e` will work? Have you tried locally?

Comment (isuruf) [reply recipe/migrations/jpeg9e.yaml:13]: Nvm, let's use a non-existing version here. `10`?

File: recipe/migrations/jpeg9e.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+migrator_ts: 1676157958
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  commit_message: "Rebuild for jpegturbo migration"
+  # limit the number of prs to do the initial test
+  pr_limit: 1
+
```
Comment (isuruf) [inline recipe/migrations/jpeg9e.yaml:9]:
```suggestion
libjpeg_turbo:
  - 2.1.5
```

File: recipe/migrations/jpeg9e.yaml (hunk 2 above)
Comment (hmaarrfk) [reply recipe/migrations/jpeg9e.yaml:13]: sure

Comment (hmaarrfk) [inline recipe/migrations/jpeg9e.yaml:13]:
```suggestion
  - 0.0
```

Comment (jakirkham) [reply recipe/migrations/jpeg9e.yaml]: Renamed the migrator filename to match this change. Hope that is ok :)

Comment (hmaarrfk) [reply recipe/migrations/jpeg9e.yaml]: Yes thank you. I suggested a change https://github.com/regro/cf-scripts/pull/1604/files

Comment (isuruf) [reply recipe/migrations/jpeg9e.yaml]: It should be something like `libjpeg_turbo.yaml`.
`jpeg9e.yaml` and `jpeg10.yaml` are both misleading.

Comment (hmaarrfk) [reply recipe/migrations/jpeg9e.yaml]: Renamed to `jpeg_to_libjpeg_turbo`

**Comment by hmaarrfk** [comment]
ok lets give this a try

**Pull request merged**